src/shapes/Object.cpp

```
@@ -0,0 +1,77 @@
+#include <sstream>
+#include <string>
+#include <tuple>
+#include <vector>
+#include "AShape.hpp"
+#include "Point3D.hpp"
+#include "Ray.hpp"
+#include "Vector3D.hpp"
+#include "Object.hpp"
+#include <iostream>
+
+std::tuple<double, Math::Vector3D, const Raytracer::IShape *> Raytracer::Object::hits(
```

Cppcheck (2.10-2)

[style] The function 'hits' is never used.

(unusedFunction)